Client/Frontend/UserContent/UserScripts/MediaBackgrounding.js

```diff
@@ -0,0 +1,143 @@
+// The below is needed because the script may not be web-packed into a bundle so it may be missing the run-once code
+
+// MARK: - Include Once
+
+if (!window.__firefox__) {
+    window.__firefox__ = {};
+}
+
+if (!window.__firefox__.includeOnce) {
+    window.__firefox__ = {};
+    window.__firefox__.includeOnce = function(key, func) {
+        var keys = {};
+        if (!keys[key]) {
+            keys[key] = true;
+            func();
+        }
+    };
+}
+
+
+window.__firefox__.includeOnce("$<MediaBackgrounding>", function() {
+    var visibilityState_Get = Object.getOwnPropertyDescriptor(Document.prototype, "visibilityState").get;
+    var visibilityState_Set = Object.getOwnPropertyDescriptor(Document.prototype, "visibilityState").set;
+    Object.defineProperty(Document.prototype, 'visibilityState', {
+        enumerable: true,
+        configurable: true,
```

Comment:
nit: Setting `configurable` to true allows scripts to delete this property. Do we need this property to be configurable?